### DISCUSSION TITLE
Random Number function

### DISCUSSION THREAD

**Discussion by vermaprk** [idaholab/moose#19516] [Q&A Modules: General] [answered]
I want to create a function which randomly takes value between 0.5 to 1. Are there such kind of inbuilt functions. Can I use ParsedFunction and provide value as random? Please help I am new to moose.

**Comment by GiudGiud** [comment, marked as answer]
@zachmprince 
I dont think we have that right now. 
FunctionIC has the code for doing this though, you could port it to a function
